README.md

```diff
@@ -6,19 +6,22 @@ Set up a local development environment for COMIT apps with one command.
 
 1. Install Docker,
 2. Install yarn or npm, 
-3. Run `yarn create comit-app` to install globally (or `npx create-comit-app` to keep it local).
-
+3. Download and unzip the latest [release](https://github.com/comit-network/create-comit-app/releases) (`zip` or `tar.gz`)
 
 ## 2 - Create your first project!
 
-1. Start local blockchain nodes & COMIT nodes: `create-comit-app start-env` (or `npx create-comit-app start-env`), 
-2. Create a new COMIT app project: `create-comit-app new my-cool-app` (or `npx create-comit-app new my-cool-app`),
-3. Open a new terminal:
-   - `cd <path-to-my-cool-app>`,
-   - Install dependencies: `yarn install` (or `npm install`),
-   - Run the [hello-swap](https://github.com/comit-network/hello-swap/) example: `yarn start` (or `npm start`),
-   - Hit `CTRL-C` once the swap is done.
-   
+1. Copy the `create-comit-app` binary to the folder where the example app should be created.
```

Comment:
I think that is confusing. I would rather instruct them to add it to their path.

Reply:
We are talking developers, hopefully they know how to execute a bin and I don;t think we should hold their hands once it's unzipped.

Reply:
Totally agree! That is why I would not tell them to put it in a specific directory but do the bare minimum thing to make our instructions correct:
- put it in a folder in $PATH
- assume it is there for the rest of the commands